Example/BatchNormTest.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  BatchNormTest.swift
+//  Example
+//
+//  Created by Mathias Claassen on 2/13/18.
+//
+
+import Accelerate
+import AVFoundation
+import MetalBender
+import MetalKit
+import MetalPerformanceShaders
+import MetalPerformanceShadersProxy
+
+struct BatchNormDataSet {
+
+    static let epsilon: Float = 0.001
+    static let mean = [Float].init(repeating: 0.5, count: 8)
```

Comment:
Maybe it's a personal taste but I prefer not writing the `init` keyword here since we almost never write it explicitly.